src/user_data.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# Here we force OpenSSH to allow the use of RSA keys.
+#
+# See https://github.com/cisagov/cool-system-internal/issues/105 for
+# more details.
+
+cat > /etc/ssh/sshd_config.d/99-allow-rsakeys.conf << ENDOFFILE
+HostKeyAlgorithms +ssh-rsa
+PubkeyAcceptedKeyTypes +ssh-rsa
```

Comment:
Per the [8.5 release notes](https://www.openssh.com/txt/release-8.5):

> ssh(1), sshd(8): rename the PubkeyAcceptedKeyTypes keyword to
   PubkeyAcceptedAlgorithms. The previous name incorrectly suggested
   that it control allowed key algorithms, when this option actually
   specifies the signature algorithms that are accepted. The previous
   name remains available as an alias. bz#3253

```suggestion
PubkeyAcceptedAlgorithms +ssh-rsa
```